### DISCUSSION TITLE
Using post-release version

### DISCUSSION THREAD

**Discussion by yyzguy** [objective-see/LuLu#629] [Q&A]
A problem I'm having was fixed in 0ad7987.  https://github.com/objective-see/LuLu/issues/608#issue-2350321633   
Is it safe to use a non-release version such as this?   Is it better to get the master release or  better to wait for next release?

**Comment by objective-see** [comment]
Any release that is not marked as a pre-release is safe to use! 
And pre-releases should be safe too (but, just haven't been as widely tested as other releases).
